services/QuillLMS/engines/evidence/spec/controllers/evidence/activated_llm_prompts_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Evidence
+  RSpec.describe ActivatedLLMPromptsController, type: :controller do
+    routes { Evidence::Engine.routes } # Use the correct engine routes
+
+    let(:json_response) { JSON.parse(response.body) }
+    let(:prompt) { create(:evidence_prompt) }
+    let(:llm_prompt) { create(:evidence_research_gen_ai_llm_prompt) }
+    let(:valid_params) do
+      {
+        research_gen_ai_activated_llm_prompt: {
+          active: true,
+          llm_prompt_id: llm_prompt.id,
+          notes: 'Sample note'
+        },
+        prompt_id: prompt.id
+      }
+    end
+
+    describe 'GET #index' do
+      let!(:activated_llm_prompt) { create(:evidence_activated_llm_prompt, llm_prompt:, prompt:, active: true) }
+
+      it 'returns a successful response' do
+        get :index, params: { prompt_id: prompt.id }
+
+        expect(response).to be_successful
+        expect(json_response['activated_llm_prompts']).to be_present
+        expect(json_response['activated_llm_prompts'].first['id']).to eq(activated_llm_prompt.id)
+      end
+
+      it 'includes associated trial data' do
+        trial = create(:evidence_research_gen_ai_trial, llm_prompt:)
```

Comment:
This is purely stylistic, so take or leave it.  I prefer to use `let` or `let!` whenever a `create` is called, but you can't call `let` inside an `it` block, so I'd structure this with a `context` wrapper.
```ruby
context 'with associated trial data' do
  let!(:trial) { create(:evidence_research_gen_ai_trial, llm_prompt:) }

  it 'includes associated trial data' do
    ... # same code
  end
end
```